### PR TITLE
Default Cardano.Testnet to NodeLoggingFormatAsJson

### PR DESCRIPTION
**Pull request by MarcFontaine**
Change `hprop_shutdownOnSlotSynced`  to parse JSON logs.

### CODE REVIEW

File: cardano-testnet/test/Test/ShutdownOnSlotSynced.hs [hunk 1]
```diff
@@ -57,15 +58,24 @@ hprop_shutdownOnSlotSynced = H.integrationRetryWorkspace 2 "shutdown-on-slot-syn
     H.cat (nodeStdout node)
     H.cat (nodeStderr node)
   mExitCodeRunning === Right ExitSuccess
+
   logs <- H.readFile (nodeStdout node)
-  slotTip <- case find (isInfixOf "Closed db with immutable tip") (reverse (lines logs)) of
```
Comment (newhoggy) [inline cardano-testnet/test/Test/ShutdownOnSlotSynced.hs:61]: This code is asserting that the node has (or will soon exit)

Is the new code asserting on the same thing?

File: cardano-testnet/test/Test/ShutdownOnSlotSynced.hs [hunk 2]
```diff
@@ -57,15 +58,24 @@ hprop_shutdownOnSlotSynced = H.integrationRetryWorkspace 2 "shutdown-on-slot-syn
...
-    Nothing -> H.failMessage callStack "Could not find current tip in node's log."
```
Comment (newhoggy) [inline cardano-testnet/test/Test/ShutdownOnSlotSynced.hs:62]: The string `"Could not find current tip in node's log."` is probably a copy-paste error.  I think it should say `"Could not find close DB message"`

**Comment by newhoggy** [comment]
There is a `TypeApplications` unused language pragma `hlint` warning.

File: cardano-testnet/test/Test/ShutdownOnSlotSynced.hs (hunk 1 above)
Comment (MarcFontaine) [reply cardano-testnet/test/Test/ShutdownOnSlotSynced.hs:61]: I updated the PR to look for `TraceOpenEvent.ClosedDB` as well.

**Pull request merged**